### PR TITLE
Feature/improve error handling

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java [hunk 1]
```diff
@@ -36,20 +37,24 @@ public static GlobalIgnoreApplier loadRecheckIgnore() {
 		try {
 			final LoadFilterWorker loadFilterWorker = new LoadFilterWorker( NopCounter.getInstance() );
 			return loadFilterWorker.load();
+		} catch ( final FileNotFoundException e ) {
+			logger.error( "Could not find recheck ignore file." );
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java:41]: We should also log the actual exception (like below):

```suggestion
			logger.error( "Could not find recheck ignore file.", e );
```

File: src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java [hunk 2]
```diff
@@ -36,20 +37,24 @@ public static GlobalIgnoreApplier loadRecheckIgnore() {
...
+		return GlobalIgnoreApplier.create( NopCounter.getInstance() );
 	}
 
 	public static GlobalIgnoreApplier loadRecheckSuiteIgnore( final File ignoreFilesBasePath ) {
 		try {
 			final LoadFilterWorker loadFilterWorker =
 					new LoadFilterWorker( NopCounter.getInstance(), ignoreFilesBasePath );
 			return loadFilterWorker.load();
+		} catch ( final FileNotFoundException e ) {
+			logger.debug( "Ignoring missing suite ignore file." );
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java:54]: We should also log the actual exception (like below):

```suggestion
			logger.debug( "Ignoring missing suite ignore file.", e );
```

**Review by beatngu13 — CHANGES_REQUESTED** [review]

File: src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java (hunk 1 above)
Comment (roesslerj) [reply src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java:41]: Actually, when you have a look at the log, (e.g. here https://travis-ci.com/retest/recheck-web/builds/112459315), not logging that exception was one of the motivations for that change. It adds a lot of noice...

File: src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java (hunk 2 above)
Comment (roesslerj) [reply src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java:54]: Same here.

**Pull request closed without merging**